src/lib/View/UserSettings/UpdateView.php

```diff
@@ -10,12 +10,15 @@
 
 use Ibexa\Core\MVC\Symfony\View\BaseView;
 use Ibexa\User\UserSetting\UserSetting;
+use Ibexa\User\UserSetting\UserSettingGroup;
 
 class UpdateView extends BaseView
 {
     /** @var \Ibexa\User\UserSetting\UserSetting|null */
     private $userSetting;
 
+    private ?UserSettingGroup $userSettingGroup = null;
```

Comment:
Nitpick: Not sure how others feel about it but I tend to set default values in the constructor.

Reply:
Unless constructor sets them to a pontentially different value, I personally prefer inline initialization. Less chance that it is forgotten and left uninitialized.

If `UserSettingsGroup` can be passed via constructor, then it's a different story and it should be initialied in constructor only.